lib/ansible/modules/system/ufw.py

```diff
@@ -245,12 +245,17 @@ def execute(cmd):
         cmd = ' '.join(map(itemgetter(-1), filter(itemgetter(0), cmd)))
 
         cmds.append(cmd)
-        (rc, out, err) = module.run_command(cmd, environ_update={"LANG": "en_EN"})
+        (rc, out, err) = module.run_command(cmd, environ_update={"LANG": "en_US"})
 
         if rc != 0:
             module.fail_json(msg=err or out)
         return (rc, out, err)
 
+    def get_current_rules():
+        return execute([[grep_bin], ["'^### tuple'"], ["/lib/ufw/user.rules"],
```

Comment:
There's no need to add additional quotes. Simply replace `"'^### tuple'"` by `'^### tuple'`.

Reply:
Not sure that work, without quote, It send me the following error message `/bin/grep: tuple: No such file or directory`

Reply:
See my new comment; that should fix this problem.